test/verify/check-active-pages

```diff
@@ -32,7 +32,7 @@ class TestActivePages(MachineCase):
         # Preloading via manifests only happens with #12534
         if m.image in [ "rhel-8-1-distropkg" ]:
             n_extra_preloaded = 0
-        elif "atomic" in m.image:
+        elif m.atomic_image or "coreos" in m.image:
```

Comment:
This could be `elif m.ostree_image`

Reply:
Ah, that was clearly a rebase error, I'll clean it up.